ambry-store/src/test/java/com.github.ambry.store/HardDeleterTest.java

```diff
@@ -256,13 +265,13 @@ public MessageInfo getMessageInfo(Read read, long offset, StoreKeyFactory factor
       tokenMovedForward = index.hardDelete();
       Assert.assertTrue(tokenMovedForward);
 
-      // indexes: [1 2] [3 4] [3d 5*] [6 7] [2d 6d] [8 9] [1d 10d] [8]
+      // indexes: [1 2] [3 4] [3d 5] [6 7] [2d 6d] [8 9*] [1d 10d] [8]
```

Comment:
The first * shows where startTokenSafeToPersist is and The second * shows where startToken/endToken are before the operations.
after every call to hardDelete, the start/end token moves by 2 blobs. the * was not moved at this line even though the token has moved